core/src/test/java/org/apache/sedona/core/serde/SedonaKryoRegistratorTest.java

```diff
@@ -0,0 +1,55 @@
+package org.apache.sedona.core.serde;
+
+import com.esotericsoftware.kryo.Kryo;
+import org.apache.log4j.Level;
+import org.apache.log4j.Logger;
+import org.apache.sedona.core.TestBase;
+import org.apache.sedona.core.serde.shape.ShapeGeometrySerde;
+import org.apache.sedona.core.serde.spatialindex.SpatialIndexSerde;
+import org.apache.spark.SparkConf;
+import org.apache.spark.api.java.JavaSparkContext;
+import org.apache.spark.serializer.KryoSerializer;
+import org.junit.*;
+import org.locationtech.jts.geom.Point;
+
+import static org.junit.Assert.assertEquals;
+
+@Ignore
+public class SedonaKryoRegistratorTest extends TestBase {
+
+    public static JavaSparkContext sc;
+    public static SparkConf conf;
+
+    @BeforeClass
+    public static void onceExecutedBeforeAll()
+    {
+        conf = new SparkConf()
+                .setAppName(SedonaKryoRegistratorTest.class.getName())
+                .setMaster("local[2]")
```

Comment:
Fix this with the "initialize" example I mentioned.

Reply:
I am using tearDown after each test that closes SparkContext so it should be fine.

I can't use the initialize because I need to init Spark by myself in order to check that the `SedonaWKBKryoRegistrator` is working.

Also, this test is [Ignored](https://github.com/apache/incubator-sedona/pull/516/files/394c496c3a879579047ca42aa807d4230694d9e1#diff-8afb6529894c10113000ff3128d33e00db24f9e122e8282f668c5be64c1d84c6R17) so it wasn't run at all at this commit.